pytest.ini

```diff
@@ -7,7 +7,6 @@
 # under the terms of the MIT License; see LICENSE file for more details.
 
 [pytest]
-pep8ignore = docs/conf.py ALL
-addopts = --pep8 --doctest-glob="*.rst" --doctest-modules --cov=invenio_oauthclient --cov-report=term-missing
+addopts = --isort --pydocstyle --pycodestyle --doctest-glob="*.rst" --doctest-modules --cov=invenio_oauthclient --cov-report=term-missing
 testpaths = docs tests invenio_oauthclient
-filterwarnings = ignore::pytest.PytestDeprecationWarning
\ No newline at end of file
+filterwarnings = ignore::pytest.PytestDeprecationWarning
```

Comment:
this was removed I believe

Reply:
You might also have to change `run-tests.sh`?

Reply:
It was suggested by @lnielsen in https://github.com/inveniosoftware/cookiecutter-invenio-module/pull/135/files